src/mlpack/methods/ann/layer/c_relu.hpp

```diff
@@ -0,0 +1,165 @@
+/**
+ * @file c_relu_impl.hpp
+ * @author Jeffin Sam
+ *
+ * Implementation of CReLU layer.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_C_RELU_HPP
+#define MLPACK_METHODS_ANN_LAYER_C_RELU_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ *
+ * A concatenated ReLU has two outputs, one ReLU and one negative ReLU,
+ * concatenated together.In other words, for positive x it produces [x, 0],
+ * and for negative x it produces [0, x].Because it has two outputs,
+ * CReLU doubles the output dimension.
+ *
+ * Note:
+ * During building of model, The next layer of crelu should have double the size
+ * as given input layer since it concatenates the input.
+ *
+ * For more information, see the following.
+ *
+ * @code
+ * @inproceedings{ICML2016,
+ *   title={Understanding and Improving Convolutional Neural Networks
```

Comment:
Do you mind to adjust the format here to be consistent with the rest of the codebase?

Reply:
@zoq , Sorry but i couldn't get it, I mean i have taken reference from your comment above, Could you just brief me what is the issue with the format

Reply:
Sure, there should be a space right before and after the `=`. Also it would be great if we could align each `=`, we can just add spaces to make it match. I think that improves the overall readability.

Reply:
@zoq , made changes , Sorry but i didn't know that the style guide was followed for this too, since the dropconnect.hpp didn't follow such, that is why i couldn't get the mistake first hand.